app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -37,4 +38,27 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
       ]
     ),
   ])
+  button_group('middleware_server_operations', [
+    select(
+      :middleware_server_power_choice,
+      'fa fa-power-off fa-lg',
+      t = N_('Power'),
+      t,
```

Comment:
`t` stands for temporary variable. It's used like this so that the N_('XXX') is written and called only once.

you can notice similar patterns elsewhere in the code in cases where you take a trivial constant and use it immediately again